ansible-ipi-install/roles/node-prep/tasks/40_bridge.yml

```diff
@@ -1,12 +1,17 @@
 ---
 - name: Setup Bridge Creation
   block:
+    - name: Get connection name using {{ prov_nic }} interface
+      shell: |
+        nmcli -f device,uuid con | awk '/{{ prov_nic }}/ {print $NF}'
+      register: prov_nic_conn
+
     - name: Delete {{ prov_nic }} due to modify nmcli bug
-      nmcli: 
-        conn_name: "{{ prov_nic }}"
+      nmcli:
+        conn_name: "{{ prov_nic_conn.stdout }}"
```

Comment:
Hey @yprokule , instead of doing the above Lets try just these changes see if you get the result your looking for

Within roles/node-prep/defaults/main.yml add the following variable:
```
prov_nic_ansible: ansible_{{ prov_nic }}
```
Within the roles/node-prep/tasks/40_bridge.yml change the delete task to the following:
```
    - name: Delete {{ prov_nic }} due to modify nmcli bug
      nmcli:
        conn_name: "{{ hostvars[ansible_fqdn][prov_nic_ansible]['device'] }}"
        type: ethernet
        state: absent
```

This should get the proper device name for the provisioning nic.

Reply:
Hey @yprokule , I played with this some more tonight. The fix is simple, we just need to add an additional delete task to check for the "System <eth>" device. Basically what happens if it doesn't find it, it just gives the ok status but it if it does it changes it. Once it deletes it and it starts re-creating the the connections it will create them without the System. 

The code should look as follows:
```
---
- name: Setup Bridge Creation
  block:
    - name: Delete {{ prov_nic }} due to modify nmcli bug
      nmcli:
        conn_name: "{{ prov_nic }}"
        type: ethernet
        state: absent

    - name: Delete System {{ prov_nic }} due to modify nmcli bug
      nmcli:
        conn_name: "System {{ prov_nic }}"
        type: ethernet
        state: absent
```
I put the beginning of the file and then the task where it should go. Can you test this in your environment see if it works as you expect? If so, can you update the PR to reflect this change?

Thanks.

Reply:
@rlopez133 that works, thanks